samples/CommunityToolkit.Maui.Sample/ViewModels/Views/AvatarViewViewModel.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Collections.ObjectModel;
+
+namespace CommunityToolkit.Maui.Sample.ViewModels.Views;
+
+/// <summary>Sample view model for Avatar View page.</summary>
+public partial class AvatarViewViewModel : BaseViewModel
+{
+#pragma warning disable CS0612 // Type or member is obsolete, this is however still used in Microsoft.Maui
```

Comment:
Let’s set the font size as a number, so we do not need to use obsolete methods

Reply:
@brminnick Is checking to see what to use, and I'll adjust accordingly  https://github.com/CommunityToolkit/Maui/pull/481#discussion_r915417622